webview/src/experiments/components/table/Row.tsx

```diff
@@ -67,7 +70,8 @@ const getMultiSelectMenuOptions = (selectedRowsList: RowProp[]) => {
     .filter(value => value.row.depth === 1)
     .map(value => value.row.values.id)
 
-  const hideRemoveOption = removableRowIds.length !== selectedRowsList.length
+  const hideRemoveOption =
+    removableRowIds.length !== selectedRowsList.length || hasRunningExperiment
```

Comment:
[C] IMO if one of the selected rows is removable we should offer the remove option and remove only the ones we can.